ksp/src/main/kotlin/com/wire/android/di/ViewModelScopedPreview.kt

```diff
@@ -76,9 +73,8 @@ internal class ViewModelScopedPreviewProcessor(private val codeGenerator: CodeGe
                 "import ${item.qualifiedName?.asString()}\n\n" +
                 "data object $name : ${item.simpleName.asString()}"
         val dependencies = Dependencies(aggregating = true, *listOfNotNull(item.containingFile).toTypedArray())
-        val file: OutputStream = codeGenerator.createNewFile(dependencies, packageName, name, "kt")
-        file.write(content.toByteArray())
-        file.close()
+        codeGenerator.createNewFile(dependencies, packageName, name, "kt")
+            .use { it.write(content.toByteArray()) }
```

Comment:
i wonder if the same is needed in the next function at line 89 ?

Reply:
You are absolutely right

Reply:
@MohamadJaara Can you have a look again, I implemented the changes :)